### ISSUE TITLE
Error in model safe / load

### ISSUE THREAD

**Issue by eule710** [apache/incubator-mxnet#1400]
Hi guys,

there seems to be an error in the model safe / load routine.

When training on cifar10 (train_cifar10.py), the following code fails to safe and load the model correctly:

```
...
model = mx.model.FeedForward(
    ctx                = devs,
    symbol             = network,
    num_epoch          = 5,#args.num_epochs,
    learning_rate      = args.lr,
    momentum           = 0.9,
    wd                 = 0.00001,
    initializer        = mx.init.Xavier(factor_type="in", magnitude=2.34),
    **model_args)

model.fit(
    X                  = train,
    eval_data          = val,
    kvstore            = kv,
    batch_end_callback = mx.callback.Speedometer(args.batch_size, 50),
    epoch_end_callback = checkpoint)

model_prefix = "model_prefix"
acts = model.predict(val)
np.savetxt("prediction1.csv", acts, fmt='%.5f', delimiter=",")    
model.save(model_prefix)

# load model

model_load = mx.model.FeedForward.load(model_prefix, 5, ctx = devs)
acts = model_load.predict(val)
np.savetxt("prediction2.csv", acts, fmt='%.5f', delimiter=",")
```

The predictions on the validation set are not the same:
prediction1.csv l.1:
0.00000,0.00000,0.00000,0.00000,0.00035,0.00000,0.00000,0.99965,0.00000,0.00000

prediction2.csv, l.1:
0.00328,0.00002,0.28973,0.00304,0.62465,0.00254,0.02564,0.00670,0.04428,0.00011

**Comment by Florro** [comment]
I experience the same problem, when I try to train a multi-stage model.

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
